### PR TITLE
Bluemix support

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#969

### CODE REVIEW

**Comment by kraman** [comment]
@see https://github.com/strongloop/strong-heroku-runner/pull/2

File: server/drivers/bluemix/executor.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+ */
+function _onInstanceExit(instanceId) {
+  var meshApp = this._server.getMeshApp();
+  var Instance = meshApp.models.ServiceInstance;
+  var self = this;
+
+  Instance.findById(instanceId, function(err, inst) {
+    if (err) {
+      return self.debug(
+        fmt('Container exited but instance not found %s', err.message)
```
Comment (rmg) [inline server/drivers/bluemix/executor.js]: It looks to me like `self.debug()` already supports format strings itself.. did you forget to remove the `fmt()`?

File: server/drivers/bluemix/executor.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+      );
+    }
+
+    inst.updateAttributes({
+      stopTime: Date.now(),
+    }, function(err) {
+      if (err) {
+        self.debug(
+          fmt('Container exited but error updating model: %s', err.message)
```
Comment (rmg) [inline server/drivers/bluemix/executor.js]: Extra `fmt()`?

File: test/bluemix-test-options.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                  "Beta. Unstable. Use at own risk."
+                ],
+                "displayName": "Beta"
+              }
+            }
+          ],
+          "dashboard_client": {
+            "id": "mJhZ9G7ppoj8d0cW7bCvF5E/u2DDuFziq+d5hGG4/fw=",
+            "secret": "YGuPSfGOS3bPSf6PxfMrUntXofgGdLAOgnHkbXWqtFg=",
+            "redirect_uri": "https://bluemix.do.strongloop.com:8701/dashboard"
```
Comment (rmg) [inline test/bluemix-test-options.json]: do? or sl?

**Comment by rmg** [comment]
@kraman this adds a fair amount of code that I suspect won't make it passed [eslint@1 + eslint-config-strongloop](https://github.com/strongloop/eslint-config-strongloop). It's all trivial stuff, but I'm wondering if you wouldn't mind taking a pass at upgrading eslint for this PR? Even if only temporarily so you can get the files added in this PR passing, it would save a lot of churn later if the new files were passing before merge.

**Comment by rmg** [comment]
Other than some nits and some eslint concerns, LGTM.

**Pull request merged**